gel/_internal/_qbmodel/_pydantic/_models.py

```diff
@@ -1240,14 +1219,9 @@ def __init__(
         wrapped = self.__proxy_of__(id, **kwargs)
         ll_setattr(self, "_p__obj__", wrapped)
 
-        ll_setattr(
```

Comment:
why this change?

Reply:
This was moved into the descriptor's `__get__`, makes more sense to do it there.

Reply:
add a comment to explain where you set the field